tm2/pkg/bft/consensus/state_test.go

```diff
@@ -1468,7 +1495,8 @@ func TestStartNextHeightCorrectly(t *testing.T) {
 	time.Sleep(5 * time.Millisecond)
 	signAddVotes(cs1, types.PrecommitType, theBlockHash, theBlockParts, vs4)
 
-	rs = cs1.GetRoundState()
+	// XXX(FIXME): this make the test hang indefinitely
+	rs = ensureGetRoundState(cs1)
```

Comment:
this is because there isn't a new round to get, it should still be in the same round, but the round state field has changed. We can just revert to the previous line.